KMReader/Services/Cache/SDImageCacheProvider.swift

```diff
@@ -39,6 +39,9 @@ enum SDImageCacheProvider {
   }()
 
   static func configureSDWebImage() {
+    // Set Accept header for image requests
+    SDWebImageDownloader.shared.setValue("image/*", forHTTPHeaderField: "Accept")
```

Comment:
SDWebImageDownloader is not use with page downloading. It's just a viewer for local file. So it should make no difference adding the header here.